modules/ocf_irc/manifests/ircd.pp

```diff
@@ -6,6 +6,12 @@
     enable    => true,
     require   => Package['inspircd'],
     subscribe => Class['ocf::ssl::default'],
+  } ->
+  cron { 'reload-irc-cert':
+    command => 'chronic /usr/local/bin/reload-ssl.sh /etc/inspircd/reload_pass',
+    hour    => 0,
+    minute  => 0,
```

Comment:
I don't think this should need to run every day, we renew certs when they have 30 days left, so could this just run weekly or something? Doesn't really matter too much though

Reply:
sure